lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb

```diff
@@ -255,6 +255,7 @@ def get_max_vrf_pipe_lengths(model)
       end
     end
     max_net_vert_distance = max_vert_distance + min_vert_distance
+    max_net_vert_distance = [max_net_vert_distance, 0.000001].max
```

Comment:
this line was deleted on master and should be deleted per a conversation with Kamel.  It was deleted on master; not sure how it got back in.

Reply:
Sorry, that what me.  I'll remove it.

Reply:
This should be fixed now.